cetmix_tower_server/readme/USAGE.md

```
@@ -1,120 +1 @@
-![Open server log](../static/description/images/server_log_usage_1.png)
-- Click the `Refresh` button to update the log. You can also click the `Refresh All` button **(3)** located above the log list in order to refresh all logs at once.
-Log output will be displayed in the HTML field below.
-
-![Update server log](../static/description/images/server_log_usage_2.png)
-
-## Using Cetmix Tower in Odoo automation
-
-You can use various Cetmix Tower functions in Odoo automation. Such as server actions, automated actions or scheduled actions.
-While you can always call any public function directly most useful helper functions are located in a special abstract model "cetmix.tower".
-You can check those functions in the source code in the following file: `models/cetmix_tower.py`
+Please refer to the [official documentation](https://cetmix.com/tower) for detailed usage instructions.
```

markdownlint-cli2 (0.17.2)

First line in a file should be a top-level heading
null

(MD041, first-line-heading, first-line-h1)